OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                            <locus from="1va" to="2rb"/>
+                            <title>Hymns</title>
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus target="#1va"/>
+                                <title ref="LIT3093RepCh373" type="complete"/>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    <locus target="#1va"/>
+                                    <hi rend="rubric">ኦእግዝእትነ፡ ቅድስት፡ ድን</hi>ግል፡ 
+                                    በ<hi rend="rubric">፪</hi><hi rend="rubric">ማርያም፡</hi> ማ<sic resp="DR">ረ</sic>ሃም፡ እሙ፡ ለኢየሱስ፡ ክርስቶስ፡ ከሀሊ<hi rend="rubric">፨</hi>
+                                    ኀበ፡ ይትነበብ፡ መጽሐፈ፡ ተአምርኪ፡ አክናፈኪ፡ ጸልሊ<hi rend="rubric">፨</hi> ወዘሰማዕነ፡ <del rend="erasure"/>
```

Comment:
Sorry for the trivial question, just for the sake of clarity: in one of our previous meeting we discussed about the punctuation signs to be adopted in our catalogue. Is ፨ included too?

Reply:
There was disagreement. @SusanneHummel was in favour of standardising to ።. I maintain that in the case of clearly identifiably ፨ (as I believe is the case here), I would prefer to write this. But it can be either substituted by one search/replace action later or possibly be visualised differently for the printed catalogue.

Reply:
Does it make a difference when doing a string search?

Reply:
if you search in oxygen, yes, but on the app punctuation either is already disregarded (?) or can be made so by Pietro (as are the homophones)